services/motion/builtin/move_request.go

```diff
@@ -16,6 +16,7 @@ import (
 	"go.viam.com/rdk/components/base/kinematicbase"
 	"go.viam.com/rdk/logging"
 	"go.viam.com/rdk/motionplan"
+	"go.viam.com/rdk/motionplan/tpspace"
```

Comment:
Is it necessary to bring this import in?  It seems like the only reason we do is to make sure that the Kinematics being returned are PTGProvider (which they really should be because this is the area of code responsible for creating it IIRC.

Reply:
it is necessary so we can determine if the planning frame is PTG or not. This then determines if inputs alone are enough to position the base where it is in the world.